pkg/ddevapp/ddevapp_test.go

```diff
@@ -538,11 +538,10 @@ func TestDdevImportDB(t *testing.T) {
 				assert.True(settingsHashSalt)
 			case "wordpress":
 				// nolint: vetshadow
-				hasAuthSalt, err := fileutil.FgrepStringInFile(app.SiteSettingsPath, "SECURE_AUTH_SALT")
+				hasAuthSalt, err := fileutil.FgrepStringInFile(app.SiteLocalSettingsPath, "SECURE_AUTH_SALT")
```

Comment:
This is odd because it's actually in our generated SiteSettingsPath, not in the SiteLocalSettingsPath (wp-config-ddev.php). Are these two files swapped somehow? I'd expect SiteSettingsPath to be wp-config.php and SiteLocalSettingsPath to be wp-config-ddev.php. BTW, I'm *way* ok with renaming that to SiteDdevSettingsPath.